packages/core/src/components/private/FormInput.tsx

```diff
@@ -21,10 +20,11 @@ export type IgnoreAttributes =
   | 'typeof'
   | 'vocab'
   // Non-standard attributes
+  | 'color'
+  | 'css'
```

Comment:
Emotions `css` prop (from Storybook) is breaking our declaration files.

Reply:
🤦‍♂